docs/reference/commandline/daemon.md

```diff
@@ -581,6 +582,12 @@ The currently supported cluster store options are:
     private key is used as the client key for communication with the
     Key/Value store.
 
+## Access authorization
```

Comment:
It isn't necessary to include all that information about how plugins are  built or where they are located in the middle of discussing how to use the option.  There is some detail that may be helpful.

```markdown
## Access authorization

Docker's access authorization can be extended by authorization plugins that your
organization can buy or build themselves. You can install one or more
authorization plugins when you start the Docker `daemon` using the
`--authz-plugins=PLUGIN_ID` option. T
```
```
$ docker daemon --authz-plugins=plugin1 --auth-plugins=plugin2,...
```
```markdown
The`PLUGIN_ID` value is either the plugin’s name or a path to its specification
file. The plugin's implementation determines whether you can specify a name or
path. Consult with your Docker administrator to get information about the
plugins available to you.

Once a plugin is installed, requests made to the `daemon` through the command
line or Docker's remote API are authorized or not by the plugin.  If you have
multiple plugins installed, at least one must authorize the request for it to
complete.

For information about how to create an authorization plugin, see [authorization
plugin](https://docs.docker.com/engine/extend/authorization/) section in the
Docker extend section of this documentation.
```